src/pages/experiments/[experimentId]/data-exploration/components/embedding/Embedding.jsx

```diff
@@ -170,12 +182,11 @@ const Embedding = (props) => {
   }
 
   const renderExpressionView = () => {
-    if (currentView.current === 'expression') {
+    if (focusData.store === 'genes') {
       return (
         <div>
-          <label htmlFor='gene name'>
-            Showing expression for gene:&nbsp;
-            <strong>{focusedGene}</strong>
+          <label htmlFor='continuous data name'>
```

Comment:
I don't see why not keeping the `gene name` title here and the `Showing expression for gene:&nbsp;` text, since there is a separate rendering text if the data is of type `cellSet`

Reply:
because it's not just gene name, it can be continuous metadata as well, which will use the same mechanic